[BUG] cannot package from develop branch due to failure in copy goal

**Describe the bug**

`mvn clean package` fails with on my machine

```
[ERROR] Failed to execute goal ch.mfrey.maven.plugin:copy-maven-plugin:1.0.0:copy (autodeploy-expath-pkgs-for-appassembler) on project exist-distribution: Execution autodeploy-expath-pkgs-for-appassembler of goal ch.mfrey.maven.plugin:copy-maven-plugin:1.0.0:copy failed: A required class was missing while executing ch.mfrey.maven.plugin:copy-maven-plugin:1.0.0:copy: org/codehaus/plexus/util/FileUtils
```

**NOTE:** This is the error that windows tests are failing with here on CI.

In order to start with freshly downloaded dependencies I ran

`mvn dependency:purge-local-repository`

This resulted in 

```
[WARNING] The POM for org.springframework:spring-webmvc:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-aop:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-beans:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-context:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-core:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-jcl:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-expression:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for org.springframework:spring-web:jar:6.0.0-M6 is missing, no dependency information available
[WARNING] The POM for io.micrometer:micrometer-observation:jar:1.10.0-M6 is missing, no dependency information available
[WARNING] The POM for io.micrometer:micrometer-commons:jar:1.10.0-M6 is missing, no dependency information available
```

**Expected behavior**

The project's develop branch can be packaged.

**To Reproduce**

If the above isn't working, please tell us the exact steps you took when you encountered the problem:
1. `mvn dependency:purge-local-repository`
2. `mvn clean package`
4. See error

**Context (please always complete the following information)**

exist-db-7.0.0-SNAPSHOT (6b13b18fc825d027b9c25424920b9f226b544208)

Apache Maven 3.9.0 (9b58d2bad23a66be161c4664ef21ce219c2c8584)
Maven home: /opt/homebrew/Cellar/maven/3.9.0/libexec
Java version: 17.0.6, vendor: Eclipse Adoptium, runtime: /Users/jll/.asdf/installs/java/adoptopenjdk-17.0.6+10
Default locale: en_DE, platform encoding: UTF-8
OS name: "mac os x", version: "12.6.3", arch: "aarch64", family: "mac"

**Additional context**
- How is eXist-db installed? built from source
- Any custom changes in e.g. `conf.xml`? none

--- a/exist-distribution/pom.xml
+++ b/exist-distribution/pom.xml
@@ -715,6 +715,18 @@
                 <configuration>
                     <showfiles>true</showfiles>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>commons-io</groupId>
+                        <artifactId>commons-io</artifactId>
+                        <version>2.11.0</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.codehaus.plexus</groupId>
+                        <artifactId>plexus-utils</artifactId>
+                        <version>1.1</version>
+                    </dependency>
+                </dependencies>
                 <executions>
                     <execution>
                         <id>autodeploy-expath-pkgs-for-appassembler</id>

